util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -192,25 +194,80 @@ private boolean isSvObWithNumberValue(McfGraph.PropertyValues node) {
   private boolean shouldExtractSeriesInfo(McfGraph.PropertyValues node) {
```

Comment:
I wonder if we should have some targeted unit tests for this functionality.

For example, change the interface of shouldExtractSeriesInfo() to take a place DCID, make it public and exercise it in unit tests...

Reply:
added in 04686d9